docs/howtos-howto.md

```diff
@@ -50,9 +50,44 @@ The workflow script for the Github action can be found at
 
 ## Resolving a conflict with an existing HOWTO
 
-TODO: This is still in progress. Currently conflicts are resolved by
-checking whether the Github action fails, and if so, manually making
-the required fixes.
+This is still a work in progress. Currently, conflicts are resolved by manually
+making the required fixes. The below can help speed up that process.
+
+### Start with a clean and updated copy of `master`
+As an example, assuming you have `https://github.com/google/flax` set as
+`upstream`:
+```bash
+git checkout master
+git fetch upstream
+git rebase upstream/master
+```
+If your local copy of `flax` isn't clean, you may need to stash your changes
+elsewhere (e.g., via `git stash`) or otherwise remove your changes.
+
+### Automatically apply as many diff hunks as possible
+The `--reject` flag tells `git apply` to apply whatever hunks in the supplied
+diff file it can and outputs the rejected hunks to a `*.rej` file in the same
+directory as the modified files (typically `examples/[EXAMPLE]`).
+
+```bash
+git apply $diff_file --reject
+```
+
+### Go through the remaining hunks manually
+This step is potentially most time-consuming. Examine the changes in each
```

Comment:
Perhaps leave out the first sentence, it sounds a bit negative and may demotivate users to try it.